Source/Core/VideoCommon/FPSCounter.h

```diff
@@ -11,20 +11,26 @@
 class FPSCounter
 {
 public:
-  // Initializes the FPS counter.
   FPSCounter();
+  ~FPSCounter();
```

Comment:
If you add a destructor, you need to add a copy-constructor, move-assignment, etc.  See [rule of 5](https://en.cppreference.com/w/cpp/language/rule_of_three)

Reply:
Sure but do we really need to if it's only ever created like this "FPSCounter m_fps_counter;"?

Reply:
> if it's only ever created like this "FPSCounter m_fps_counter;"

It's just a good way to bullet proof code, even if the possibility is very slim.

Reply:
Seems like a small nitpick to me, I have loads of PRs like this to submit yet, can't satisfy everyone :).

Reply:
If you don't actually need the copy constructor etc, feel free to mark them as ` = delete` instead of implementing them. That way nobody will accidentally use the auto-generated ones.

Reply:
Is it fine now? I marked it as solved just in case no one answered.

Reply:
Yep, that looks fine.